chromium_src/components/signin/public/identity_manager/identity_manager.h

```diff
@@ -6,8 +6,15 @@
 #ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_SIGNIN_PUBLIC_IDENTITY_MANAGER_IDENTITY_MANAGER_H_
 #define BRAVE_CHROMIUM_SRC_COMPONENTS_SIGNIN_PUBLIC_IDENTITY_MANAGER_IDENTITY_MANAGER_H_
 
-#define GetAccountsInCookieJar           \
-  GetAccountsInCookieJar_Unused() const; \
+#define GetAccountsInCookieJar                                        \
```

Comment:
presubmit gives me a message, probably related to this file:
```
** Presubmit Messages: 1 **
-------------------------
[Info] chromium_src:
Located 10 #define statements.
```

Reply:
This is just an informational message. Nothing needs to be done here.